.buildkite/pipeline.yml

```diff
@@ -24,9 +24,16 @@ steps:
   - label: "🔬 Validate Podspecs"
     key: "validate"
     command: |
-      validate_podspec
+      # validate_podspec
+      echo '+++ ⚠️ validate_podspec was bypassed ⚠️'
+      # post a message in the logs
+      cat .buildkite/validate_podspec_annotation.md
+      # and also as an annotation
+      cat .buildkite/validate_podspec_annotation.md | buildkite-agent annotate --style 'warning'
```

Comment:
There is a new workaround for this issue: You can pass a `--patch-cocoapods` option to `validate_podspec` and the publish pod utilities. See https://github.com/wordpress-mobile/WordPressKit-iOS/pull/652/files.